src/Polytopes/LinearProgram.jl

```diff
@@ -0,0 +1,147 @@
+@doc Markdown.doc"""
+    LinearProgram(P, c [, k])
+
+The linear program on the feasible set P (a Polyhedron) with
+ respect to the function x ↦ dot(c,x)+k where k is optional (default 0).
+
+""" struct LinearProgram
+   feasible_region::Polyhedron
+   polymake_lp::Polymake.BigObjectAllocated
+   convention::Symbol
+   function LinearProgram(Q::Polyhedron, objective::AbstractVector, k; convention = :max)
+      P=Polyhedron(Polymake.polytope.Polytope(pm_polytope(Q)))
+      ambDim = ambient_dim(P)
+      size(objective, 1) == ambDim || error("objective has wrong dimension.")
+      lp = Polymake.polytope.LinearProgram(LINEAR_OBJECTIVE=homogenize(objective, k))
+      pm_polytope(P).LP = lp
+      new(P, lp, convention)
+   end
+   function LinearProgram(Q::Polyhedron, objective::AbstractVector; convention = :max)
+      LinearProgram(Q,objective,0; convention=convention)
+   end
```

Comment:
Can simplify this by using a default value for `k`.
```suggestion
   function LinearProgram(Q::Polyhedron, objective::AbstractVector, k = 0; convention = :max)
      P=Polyhedron(Polymake.polytope.Polytope(pm_polytope(Q)))
      ambDim = ambient_dim(P)
      size(objective, 1) == ambDim || error("objective has wrong dimension.")
      lp = Polymake.polytope.LinearProgram(LINEAR_OBJECTIVE=homogenize(objective, k))
      pm_polytope(P).LP = lp
      new(P, lp, convention)
   end
```

Reply:
Also, perhaps restrict the type of `k`? In `homogenize` it is `Number`. I have no idea what it does, so I can't say if that makes sense; for me, `k` sounds as if it was an integer, so perhaps `Integer` or even `Int` might make more sense? On the other hand, perhaps an `fmpz` should be allowed, too? (but of course that doesn't work right now, as `homogenize` won't accept it)